app/src/ui/history/commit-list.tsx

```diff
@@ -36,7 +62,7 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     return (
       <CommitListItem
         key={commit.sha}
-        gitHubRepository={this.props.repository.gitHubRepository}
+        gitHubRepository={this.props.gitHubRepository}
```

Comment:
This makes the dependency more explicit, because carrying around the full `repository` model is something we should avoid unless it's required.